testdata/go/v3/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -146,8 +147,8 @@ spec:
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 100m
-                    memory: 30Mi
+                    cpu: 200m
```

Comment:
This is a user facing change but not breaking. Do we need migration guide?

Reply:
Yep, and note all operator types affected in the header.